### ISSUE TITLE
Edit Item - code/handle

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/RCO#956]
In `ItemEdit`, if I change code/cat/handle, the new values are not stored.

I've watched the `Network` tab, and don't see `put/post` to the relevant `item` REST end-points.

**Comment by TahaKhanAbdalli** [comment]
Hello Ian,
I've examined the workflow for editing the code, cave, or handle, and I've identified an issue related to deletion. Let me clarify how we currently handle the editing and updating of values in RCO.
When we edit the value of code, cave, or handle, a new array is created. This array comprises both the new and old values. Upon pressing the save button, the system deletes the array that exclusively contains the old values of code, cave, or handle that were edited. It then updates it with the new array.
The problem arises because our application doesn't allow deletion. Consequently, it fails to delete the previous array containing the old values and doesn't properly update the new array, which includes both old and new values. This restriction on deletion is preventing us to update the values of code, cave or handle.
